selendroid-test-app/build.gradle

```diff
@@ -3,44 +3,43 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        // https://bintray.com/android/android-tools/com.android.tools.build.gradle/view
+        classpath 'com.android.tools.build:gradle:1.4.0-beta4'
     }
 }
 repositories {
     jcenter()
 }
-apply plugin: 'com.android.library'
+
+apply plugin: 'com.android.application'
 
 dependencies {
-    compile group: 'commons-io', name: 'commons-io', version:'2.2'
-    compile group: 'commons-logging', name: 'commons-logging', version:'1.1.2'
-    compile group: 'org.json', name: 'json', version:'20090211'
-    compile group: 'net.sf.saxon', name: 'Saxon-HE', version:'9.4.0.6'
-    compile group: 'dom4j', name: 'dom4j', version:'1.6.1'
-    compile group: 'xml-apis', name: 'xml-apis', version:'1.4.01'
-    compile group: 'org.apache.httpcomponents', name: 'httpclient', version:'4.3.4'
-    compile "junit:junit:4.8.2"
-    compile group: 'org.hamcrest', name: 'hamcrest-all', version:'1.3'
-    compile group: 'org.mockito', name: 'mockito-all', version:'1.9.5'
-    compile group: 'com.google.android', name: 'android', version:'2.3.3'
-
-    compile files('../third-party/cordova-3.7.0/classes.jar')
-    compile files('../third-party/cordova-4.0.0/classes.jar')
-    compile files('../third-party/support-v4/classes.jar')
-    compile project(':selendroid-standalone')
-    compile project(':selendroid-client')
-    compile project(':selendroid-server-common')
+    testCompile 'junit:junit:4.8.2'
+    testCompile 'org.hamcrest:hamcrest-all:1.3'
+    testCompile 'org.mockito:mockito-all:1.9.5'
+
+    compile 'com.android.support:support-v4:23.0.1'
 }
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1" // https://developer.android.com/tools/revisions/build-tools.html
 
     defaultConfig {
         minSdkVersion 10
         targetSdkVersion 16
     }
 
+    adbOptions {
+        timeOutInMs 60 * 1000
```

Comment:
The default ADB timeout is very low so we have to manually increase it.
https://code.google.com/p/android/issues/detail?id=104322